### DISCUSSION TITLE
Utility to merge schemas (f.e. allOf)?

### DISCUSSION THREAD

**Discussion by glen-84** [microsoft/OpenAPI.NET#1629] [Q&A] [answered]
Is there a utility method to merge multiple schemas into one, for example if you needed to merge the `allOf` schemas?

**Comment by Rujith** [comment]
Have you seen Redocly CLI ? https://redocly.com/docs/cli/commands/join/

**Reply by glen-84** [reply]
I was asking about merging `OpenApiSchema` objects, not API documents.

**Reply by baywet** [reply, marked as answer]
This is currently not supported by the library. Please open an issue if you'd like this feature to be added so it can be discussed further.
